rhdm70-image-streams.yaml

```diff
@@ -11,9 +11,11 @@ items:
   metadata:
     name: rhdm70-decisioncentral-openshift
     annotations:
+      openshift.io/image.insecureRepository: 'true'
```

Comment:
IMHO, it should be added in the upstream.
You can use it for local development, when the image gets released it should be available in the public secured registry.

Reply:
Sry, it shouldn’t be added on upstream.

Reply:
Agreed!
Let me put this in the comment status, so it's easy for a user to open it in dev mode.